src/main/java/org/sitenv/ccdaparsing/model/CCDAMedicalEquipmentOrg.java

```diff
@@ -0,0 +1,77 @@
+package org.sitenv.ccdaparsing.model;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
```

Comment:
You might want your comment here like the other classes to identify it was your work?